dataplane/Dockerfile

```
@@ -1,26 +1,42 @@
-FROM archlinux as builder
+FROM ubuntu:23.10 as builder
 
-RUN pacman -Syu --noconfirm
-RUN pacman -S base-devel protobuf rustup --noconfirm
+ARG TARGETARCH
 
+RUN apt update
+RUN apt install --yes build-essential protobuf-compiler curl pkg-config llvm
```

If I'm not remembering wrong, `apt` is the frontend for a human admin, and `apt-get` is meant for things like automated use cases?

```suggestion
RUN apt-get update
RUN apt-get install --yes build-essential protobuf-compiler curl pkg-config llvm
```